### PR TITLE
fix(docs): update broken links in multiple sections of docs

### PR DESCRIPTION
**Pull request by Shalinit3**
### Description
Will fix the broken links of examples and config options in the getting started section of docs.
Will fix the auth-recipes link in examples Readme.

### Check List
If not relevant to pull request, check off as complete

- [x] All tests passing
- [x] Docs updated with any changes or examples if applicable
- [x] Added tests to ensure new feature(s) work properly

### CODE REVIEW

File: examples/complete/firestore/README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Your app is ready to be deployed!
 
 ### Integrating Auth
 
-Checkout [the auth recipes](/docs/recipes/auth) for some simple examples of how to integrate auth.
```
Comment (prescottprue) [inline examples/complete/firestore/README.md]: This link actually already works for me in both the gitbook and markdown versions of the docs. Does it not work for you? If not, how are you viewing?

File: examples/complete/simple/README.md [hunk 2]
```diff
@@ -65,6 +65,6 @@ Your app is ready to be deployed!
 
 ### Integrating Auth
 
-Checkout [the auth recipes](/docs/recipes/auth) for some simple examples of how to integrate auth.
```
Comment (prescottprue) [inline examples/complete/simple/README.md]: Same with this one, works for me in both versions.

File: docs/getting_started.md [hunk 3]
```diff
@@ -88,7 +88,7 @@ export default compose(
 
 ### Decorators
 
-They are completely optional, but ES7 Decorators can be used. [The Simple Example](examples/simple) shows implementation without decorators, while [the Decorators Example](examples/decorators) shows the same application with decorators implemented.
+They are completely optional, but ES7 Decorators can be used. [The Simple Example](https://github.com/prescottprue/react-redux-firebase/blob/master/examples/complete/simple/src/Home.js) shows implementation without decorators, while [the Decorators Example](https://github.com/prescottprue/react-redux-firebase/blob/master/examples/snippets/decorators/App.js) shows the same application with decorators implemented.
```
Comment (prescottprue) [inline docs/getting_started.md]: Great catch with these. We should be able to just add the preceding slash though right?

Maybe we should keep the references to the README of those examples and explain more in the readme?

File: examples/complete/firestore/README.md (hunk 1 above)
Comment (Shalinit3) [reply examples/complete/firestore/README.md]: No, not working for me. These are in the Readme of the examples.

Comment (Shalinit3) [reply examples/complete/firestore/README.md]: broken-links

File: docs/getting_started.md (hunk 3 above)
Comment (Shalinit3) [reply docs/getting_started.md]: No, I tried adding slashes but it didn't work. Even some other relative links that are not mentioned in the Readme of the docs are also broken.

File: examples/complete/firestore/README.md (hunk 1 above)
Comment (prescottprue) [reply examples/complete/firestore/README.md]: Weird - Thanks for the screen capture.

You may already know this, but the gitbook rendered version of the docs is available at [react-redux-firebase.com](https://react-redux-firebase.com).

Comment (Shalinit3) [reply examples/complete/firestore/README.md]: These links are in the Readme of the complete examples- simple & firestore.

**Pull request merged**